utils/ansible-runner-service.sh

```diff
@@ -414,17 +414,20 @@ check_access() {
 }
 
 manage_ansible_hosts_file() {
-    echo "Linking the default ansible hosts file to the runner-service inventory."
+    echo "Linking the runner service inventory to ceph-ansible hosts"
+    ln -sf /usr/share/ceph-ansible/hosts /usr/share/ansible-runner-service/inventory/hosts
+
+    echo "Linking the default ansible hosts file to the ceph-ansible hosts file."
     if [ -h "$DEFAULT_ANSIBLE_HOSTS" ]; then
```

Comment:
we probably need to reassess this. we were linking to the default ansible hosts file - now we're not, so the logic here needs review.

We should be 
creating /usr/share/ceph-ansible/hosts - if it doesn't exist
linking /usr/share/ansible-runner-service/inventory/hosts to /usr/share/ceph-ansible-hosts
checking to see if the /usr/share/ceph-ansible/hosts contains hosts - if so, we should move it to one side, tell the admin and set up a new one to use with the installer. (remember ansible supports yaml, and ini format inventory files - so we need to get the inventory to a known state here)

we should no longer be manipulating /etc/ansible/hosts - wither directly or via symlink